ephemeralkey.go

```diff
@@ -0,0 +1,42 @@
+package stripe
+
+import "encoding/json"
+
+// EphemeralKeyParams is the set of parameters that can be used when creating
+// an ephemeral key.
+// For more details see https://stripe.com/docs/api#ephemeral_keys.
+type EphemeralKeyParams struct {
+	Params
+	Customer string
+}
+
+// EphemeralKey is the resource representing a Stripe ephemeral key.
+// For more details see https://stripe.com/docs/api#ephemeral_keys.
+type EphemeralKey struct {
+	ID      string `json:"id"`
+	Created int64  `json:"created"`
+	Expires int64  `json:"expires"`
+	Live    bool   `json:"livemode"`
+    AssociatedObjects []struct {
```

Comment:
You may not have been told, but the Go people _really_ like tabs :)

You should probably change this in your editor (there's some good Go plugins for Vim if you're on that), but the toolchain also ships with a tool called `gofmt` to fix this stuff for you:

```
gofmt -w *.go
```

We have a check in CI on this, so you'll need to fix it before the build will pass.